src/ontology/cl-edit.owl

```diff
@@ -32727,6 +32729,26 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:7107724") oboInOwl:hasRe
 AnnotationAssertion(rdfs:label obo:CL_4052011 "follicle associated enterocyte")
 SubClassOf(obo:CL_4052011 obo:CL_0000584)
 
+# Class: obo:CL_4052015 (endocrine gland capillary endothelial cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:18480313") obo:IAO_0000115 obo:CL_4052015 "Any capillary endothelial cell that is part of the endocrine gland.")
+AnnotationAssertion(terms:contributor obo:CL_4052015 <https://orcid.org/0009-0000-8480-9277>)
+AnnotationAssertion(terms:date obo:CL_4052015 "2024-09-30T15:27:49Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:CL_4052015 "endocrine gland capillary endothelial cell")
+SubClassOf(obo:CL_4052015 obo:CL_0002144)
+SubClassOf(obo:CL_4052015 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0002368))
```

Comment:
Given the text def, this looks like a grouping class so should have an EquivalentClass definition.